db/migrate/20170428195326_replace_course_evaluation.rb

```diff
@@ -0,0 +1,18 @@
+class ReplaceCourseEvaluation < ActiveRecord::Migration
+  def up
+    Questionnaire.where(type: "CourseEvaluationQuestionnaire"){
+        | questionnaire |
```

Comment:
If the block is multiline, it will be better to write the block like this
```
CourseEvaluationQuestionnaire.all do | questionnaire |
[code]
end
```